grid/client.py

```diff
@@ -104,38 +104,81 @@ def destroy(self):
 
     @property
     def models(self, N: int = 1):
-        models = json.loads(self._send_get("models/", N=N))["models"]
-        return models
+        return json.loads(self._send_get("models/", N=N))
+
+    def delete_model(self, model_id):
+        return json.loads(
+            self._send_post(
+                "delete_model/", data={"model_id": model_id}, unhexlify=False
+            )
+        )
+
+    def serve_model(self, model, model_id=None, is_private_model=False):
+        """Hosts the model and optionally serve it using a Rest API.
+
+        Args:
+            model: A jit model or Syft Plan.
+            model_id: An integer or string representing the model id used to retrieve the model
+                later on using the Rest API. If this is not provided and the model is a Plan
+                we use model.id, if the model is a jit model we raise an exception.
+            is_private_model: A boolean indicating if the model is private or not. If the model
+                is private the user does not intend to serve it using a Rest API due to privacy reasons.
+
+        Returns:
+            None if is_private_model is True, otherwise it returns a json object representing a Rest API response.
```

Comment:
Couldn't we return an unified type of result? Maybe we can handle the different responses (websocket /rest) inner the method and return a unified result.

Reply:
Just an idea :).

Reply:
I agree, we should definitely standardize return types. But I think we could do this in a different PR, this is already bigh enough. Is this okay?

Reply:
Ok.